semantic-conventions/README.md

```diff
@@ -2,6 +2,23 @@
 
 A docker image to process Semantic Convention YAML models.
 
+## Development
```

Comment:
I think these instructions are great, but I wonder if they aren't better suited for a `CONTRIBUTING.md`? I think the main README is better suited for people coming here and wanting to learn how to use it only to generate the conventions.